index.js

```diff
@@ -58,12 +58,19 @@ function delay(ms){
 
 class Challenge {
 	constructor(options = {}){
+		this.module = "acme-dns-01-cloudflare";
 		this.options = options;
-		this.client = new cloudflare({
-			email: options.email,
-			key: options.key,
-			token: options.token
+		this.cfClient = new cloudflare({
+			email: options.client?.email || options.email,
+			key: options.client?.key || options.key,
+			token: options.client?.token || options.token
 		});
+		this.client = {
+
+			email: options.cfClient?.email || options.client?.email || options.email,
+			key: options.cfClient?.key || options.client?.key || options.key,
+			token: options.cfClient?.token || options.client?.token || options.token
+		}
```

Comment:
If you fix up this minor issue flagged by ESLint, that would be great!

Reply:
Done!